api/cards/filterCards.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from "express";
+import { filterKnownCards, getAllCards } from "~/utils/cards";
+
+const knownTags: any = {};
+
+export default (_: Request, response: Response) =>
+  getAllCards()
+    .then(cards => filterKnownCards(cards))
+    .then(cards => {
+      cards.forEach(card => {
+        card.tags.forEach(tag => {
+          if (!knownTags[tag]) {
+            knownTags[tag] = 1;
+          } else {
+            knownTags[tag] += 1;
```

Comment:
Do you think it's necessary to count all the tag's amount in the repo?

Reply:
what's the point to have this counter?

Reply:
For user experience - user would like to know how many different tags are here. Because we don't have any collected info about what we have at the repo.

Reply:
we can create aggregated info in cards, if we need